docs/pages/modules/get-started.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Get Started
+---
+
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
+
+## Create a new module
+
+To create a new Expo module from scratch, just run `yarn create expo-module` or `npm create expo-module`.
+The script will ask you a few questions and then generate the native Expo module along with the example app for iOS and Android that uses your new module.
+
+## Set up a library as an Expo module
+
+You may want to use the Expo Modules API in existing React Native libraries, for example with [AppDelegate Subscribers](./appdelegate-subscribers.md) you can hook into `AppDelegate` methods without requiring developers to copy any code over to their own `AppDelegate`. This is particularly useful to add seamless support for Expo managed projects to a library. The following steps will set up your existing React Native library to have access to the Expo Modules API.
+
+Create the [module config](module-config) **expo-module.config.json** file just near your **package.json** and start from the empty object `{}` in there. We will fill it in later to enable specific features. <br/>
```

Comment:
```suggestion
Create the [module config](module-config) **expo-module.config.json** file just near your **package.json** and start from the empty object `{}` in there. We will fill it in later to enable specific features.

```

Empty line in MD should replace the `<br/>` if I understand the use case correcctly.